tasks/suid_sgid.yml

```diff
@@ -20,5 +20,5 @@
 - name: remove suid/sgid bit from all binaries except in system and user whitelist
   file: path='{{item}}' mode='a-s' state=file follow=yes
   with_items:
-    - '{{ suid | default(omit) | difference(os_security_suid_sgid_whitelist) }}'
+    - '{{ suid | default() | difference(os_security_suid_sgid_whitelist) }}'
```

Comment:
In this case better use `with_flattened` and `default([])` :) Similar goes with the other uses of `default` in `with_items`.